superset/assets/package.json

```diff
@@ -118,6 +119,7 @@
     "react-dom": "^16.4.1",
     "react-gravatar": "^2.6.1",
     "react-hot-loader": "^4.3.6",
+    "react-json-tree": "^0.11.2",
```

Comment:
We use `react-syntax-highlighter` in other places. I guess the tree is nicer as far as collapsing/expanding. As a downside though, it looks like `react-json-tree` may not copy/paste well though (from the screenshot) and not sure whether the colors match our theme (no big deal, but wanted to highlight it).

Reply:
I’ve added a copy button to the dialog too. Will update screenshots tomorrow

Reply:
Dialog screenshot updated. I could also look into building a custom theme based off the colors we already use, but i'm not sure if it matters that much

Reply:
Let's not worry about colors at the moment